koala/cogs/Voting.py

```diff
@@ -19,7 +19,7 @@
 import KoalaBot
 
 # Constants
-logging.basicConfig(filename=KoalaBot.CONFIG_DIR+'Vote.log')
+# logging.basicConfig(filename=KoalaBot.CONFIG_DIR+'Vote.log')
```

Comment:
Why not just delete this?

Reply:
To be replaced with log.logger with rest of refactor